.github/workflows/release.yml

```diff
@@ -38,21 +36,21 @@ jobs:
         python-version: ['3.5', '3.6', '3.7']
       fail-fast: false
     steps:
-      - if: github.event_name == 'schedule'
+      - if: github.event_name == 'push'
```

Comment:
So i believe this will trigger on release branches as well. Make it push && master?

Reply:
Nice catch. Let me change that.